### DISCUSSION TITLE
Is Carlinkit adapter required for wired CarPlay?

### DISCUSSION THREAD

**Discussion by isaac-mcfadyen** [tesla-android/issue-tracker#52] [Q&A] [answered]
In the Install gude, it says that the Carlinkit adapter 'enables Apple CarPlay or Android Auto."

However, in the Hardware Requirements section it states that it is required (emphasis mine) "if you intend to use **wireless** CarPlay or Android Auto."

Is wired CarPlay supported with a Lightning-to-USB (without a dongle)?

Sorry if this is an obvious question.

**Comment by rrlevy** [comment, marked as answer]
You need the Carlinkit adapter, even for wired carplay!

**Reply by isaac-mcfadyen** [reply]
Alright, thanks! I'll PR the docs to clarify.

EDIT: PR here https://github.com/tesla-android/tesla-android.github.io/pull/26

**Reply by focher** [reply]
I would also recommend to update the installation documentation on the main website. Currently, the CarlinKit CPC200-Autokit, CPC200-CCPA is listed under "Optional" components in the Hardware Requirements but it's really required.

**Reply by adamChrystie** [reply]
The carlinkit dongle is listed as optional because not everyone wants CarPlay functionality.

**Comment by ch7883j22jdiyhjjj** [comment]
Is the Carlinkit the only usable CarPlay adapter or can I use cplay2air?

**Reply by mikegapinski** [reply]
Only cpc200-ccpa. The other one you mentioned converts wired CarPlay into wireless on OEM headunits from VW or any other manufacturer

**Reply by ch7883j22jdiyhjjj** [reply]
Thank you
